invokeai/frontend/web/src/features/parameters/components/MainModel/ParamMainModelSelect.tsx

```diff
@@ -42,19 +43,27 @@ const ParamMainModelSelect = () => {
     });
 
   return (
-    <InvControl
-      label={t('modelManager.model')}
-      isDisabled={!options.length}
-      isInvalid={!options.length}
+    <InvTooltip
+      label={
+        Object.values(Object.values(data?.entities ?? {})).find(
+          (m) => m.model_name === model?.model_name
+        )?.description
```

Comment:
- This should be in a `useMemo`, so there is less logic in the render fn, and to reduce the number of times we need to search through the query data.
- There is a hidden bug. Model name is not a unique identifier. This will be resolved in the upcoming MM refactor but until then we need to account for it.
- We can leverage RTK features to avoid the bug and also make the intent clearer.

```ts
const tooltipLabel = useMemo(() => {
  if (!data || !model) {
    return;
  }
  return mainModelsAdapterSelectors.selectById(data, getModelId(model))
    ?.description;
}, [data, model]);
```